src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -829,16 +829,6 @@ FixedwingPositionControl::control_position(const math::Vector<2> &curr_pos, cons
 				wp_distance_save = 0.0f;
 			}
 
-			// create virtual waypoint which is on the desired flight path but
```

Comment:
That's definitely good practice, although in this case I think SITL is sufficient.